### DISCUSSION TITLE
本科论文摘要目录加上罗马页码

### DISCUSSION THREAD

**Discussion by LINZsss** [whutug/whu-thesis#215] [Q&A]
求问写本科毕业论文时，怎么在摘要目录处加上罗马页码

**Comment by xkwxdyy** [comment]
> 求问写本科毕业论文时，怎么在摘要目录处加上罗马页码

为什么要加，是学院还是哪里的要求吗

**Reply by LINZsss** [reply]
是学院审核后的要求

**Reply by xkwxdyy** [reply]
有没有相关信息的截图

**Reply by LINZsss** [reply]
![IMG_20230523_230801](https://github.com/whutug/whu-thesis/assets/100743457/c98b34e4-e0fc-473e-b9a5-0c125a242f93)

**Reply by LINZsss** [reply]
学院检查后要求补充页码

**Reply by xkwxdyy** [reply]
他没说放在哪里，什么字号

**Reply by LINZsss** [reply]
是的，审核老师应该是习惯性的拿硕士论文的要求

**Comment by SwitWu** [comment]
https://github.com/whutug/whu-thesis/blob/6f19c2e0d34bbfc752f86a7ea505154a185ff595/whu-thesis.cls#L1065-L1077

相关代码在这，如果有强制需求，可以自行在 1069 行前面加 
```
\pagestyle{plain}
\pagenumbering{Roman}
```
